sections/homepage-getting-started.js

```diff
@@ -36,141 +37,75 @@ const SecondButton = styled.button`
-
-      \`\`\`jsx
-      import styled from 'styled-components';
-
-      const Button = styled.button\`\`;
-      \`\`\`
-
-      This \`Button\` variable here is now a React component that you can use like any other React component!
-      This unusual backtick syntax is a new JavaScript feature called a tagged template literal.
-      You know how you can call functions with parenthesis? (\`myFunc()\`) Well, now you can also call functions with backticks!
-      [here is an explanation](/docs/advanced#tagged-template-literals) of how that works exactly.
-
-      If you render our lovely component now (just like any other component: \`<Button />\`) this is what you get:
-
-      ${
-      <AlignCenter>
-        <button>I'm a &lt;Button /&gt;!</button>
-      </AlignCenter>
-      }
-
-      It renders a button! That's not a very nice button though 😕 we can do better than this,
-      let's give it a bit of styling and tickle out the hidden beauty within!
-
-      \`\`\`jsx
-      const Button = styled.button\`
-        border-radius: 3px;
-        padding: 0.25em 1em;
-        margin: 0 1em;
-        background: transparent;
-        color: palevioletred;
-        border: 2px solid palevioletred;
-      \`;
-      \`\`\`
-
-      ${
-      <AlignCenter>
-        <SecondButton>I'm a styled &lt;Button /&gt;</SecondButton>
-      </AlignCenter>
+  <I18n
+    ns="homeGettingStarted"
+    wait={process.browser}
+  >
+    {
+      (translate) => {
+        if (!translate) {
+          return null
+        }
+
+        return (
+          <Content>
+            <AlignCenter>
+              <a href="https://github.com/styled-components/styled-components">
+                <Badge
+                  src="https://img.shields.io/github/stars/styled-components/styled-components.svg?style=social&label=Star&maxAge=3600"
+                  alt={translate('starsOnGithub')}
+                />
+              </a>
+              <a href="https://www.npmjs.com/package/styled-components">
+                <Badge
+                  src="https://img.shields.io/npm/v/styled-components.svg"
+                  alt={translate('currentVersion')}
+                />
+              </a>
+              <Badge
+                src="https://img.shields.io/npm/dm/styled-components.svg?maxAge=3600"
+                alt={translate('monthlyDownloads')}
+              />
+              <Badge
+                src="http://img.badgesize.io/https://unpkg.com/styled-components/dist/styled-components.min.js?compression=gzip&label=gzip%20size"
+                alt={translate('gzipSize')}
+              />
+              <a href="https://spectrum.chat/styled-components">
+                <Badge
+                  src="https://withspectrum.github.io/badge/badge.svg"
+                  alt={translate('joinTheCommunity')}
+                />
+              </a>
+            </AlignCenter>
+
+            {md(translate('markdown.first'))}
+            <AlignCenter>
```

Comment:
Here is where I had to render <AlignCenter>, <ExampleButton>, etc. directly in the component. As I couldn't use js interpolation. Which makes sense, we don't need that in the translation file.

Reply:
Take a look @philpl 😄

Reply:
I'm still wondering whether we should just split the file and make one per translation for content at least, like this and the docs pages :)

Reply:
Good question. I'd suggest we split it per page, so we have a translation file per component page. As it's intuitive how to find the translations. The downside to that is that we might end up with a lot of translation files and keys and that could be hard to manage. So I'll leave that to decide @philpl and @mxstbr

Reply:
@philpl So I went back to check the documentation. Unfortunately we're stuck using `namespaces` (`ns`), and the API only provides this as customization
```js
backend: {
  loadPath: __dirname + '/locales/{{lng}}/{{ns}}.yml'
}
```
So I think we can't have `locales/en/docs/first.yml` 😞

Reply:
You can take a look [here](https://github.com/i18next/i18next-node-remote-backend)